Data/Hashable.hs

```diff
@@ -240,3 +246,14 @@ instance (IsString a, Hashable a) => IsString (Hashed a) where
 instance Foldable Hashed where
   foldr f acc (Hashed a _) = f a acc
 
+-- | 'Hashed' cannot be 'Functor'
+mapHashed :: Hashable b => (a -> b) -> Hashed a -> Hashed b
+mapHashed f (Hashed a _) = hashed (f a)
+
+-- | 'Hashed' cannot be 'Traversable'
+traverseHashed :: (Hashable b, Functor f) => (a -> f b) -> Hashed a -> f (Hashed b)
+traverseHashed f (Hashed a _) = fmap hashed (f a)
+
+-- | 'Hashed' cannot be 'Monad'
+joinHashed :: Hashed (Hashed a) -> Hashed a
```

Comment:
Well, I don't know, added it for "completeness". Not insisting on it to be included.